### ISSUE TITLE
Taxonomy Terms Could Be Collasped

### ISSUE THREAD

**Issue by yqzhen1990** [OrchardCMS/OrchardCore#16770]
### Is your feature request related to a problem? Please describe.
When you have a hierarchy of lots of terms , you need to scroll and scroll and scroll or use "Ctrl+F" to find the term you want. 

### Describe the solution you'd like
I'd like the parent term with sub terms could be collasped to have a better experience.

**Comment by hishamco** [comment]
Please show me a screenshot for an example where you want to use collapsable items

**Comment by yqzhen1990** [comment]
> Please show me a screenshot for an example where you want to use collapsable items

![image](https://github.com/user-attachments/assets/4fb910ca-fb02-4b15-9225-91b4400f8ba5)

It could be better , if the terms can be collasped.

**Comment by hishamco** [comment]
This is too much, I didn't expect such a number :)

We might need to improve the UI a little bit

**Comment by yqzhen1990** [comment]
> This is too much, I didn't expect such a number :)
> 
> We might need to improve the UI a little bit

Yes, some industrial products have many categories and levels, and I met this situation.

**Comment by baipai** [comment]
@yqzhen1990 Hey, we're currently working on improving the admin experience based on #17327 . Right now, we’re collecting survey responses to better understand people’s workflows, pain points, and where the experience can be improved, and to use that insight to inform future improvements.

The points you raised in this issue and also in [#16658](https://github.com/OrchardCMS/OrchardCore/issues/16658), highlight some important perspective we'd love to follow up on. If you're open to it, we'd really appreciate you taking a few minutes to fill out the survey. Your insights would be incredibly valuable.

Survey: https://forms.gle/cXsvwkZfco65ud3o6
